src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -548,6 +548,496 @@ type Address @model {
 }
 ```
 
+### Relational models with custom primary keys
```

Comment:
@renebrandel - this is new, could you have a look? Thanks!

Reply:
Discussed offline, @AaronZyLee has [a PR](https://github.com/aws-amplify/docs/pull/4532) out to update this section.